src/pyconverter/xml2py/writer.py

```diff
@@ -295,90 +391,246 @@ def write_source(
+        #     class_structure = {}
+        #     for initial_class_name, method_list in class_map.items():
+        #         additional_imports = None
+        #         if initial_class_name in specific_classes.keys():
+        #             specific_class_dict = specific_classes[initial_class_name]
+        #             class_name = specific_class_dict["class_name"]
+        #             file_name = specific_class_dict["file_name"]
+        #             file_path = os.path.join(module_path, f"{file_name}.py")
+        #             if "additional_imports" in specific_class_dict.keys():
+        #                 additional_imports = specific_class_dict["additional_imports"]
+        #         else:
+        #             class_name = initial_class_name.title().replace(" ", "").replace("/","")
+        #             file_name = initial_class_name.replace(" ", "_").replace("/","_").lower()
+        #             file_path = os.path.join(module_path, f"{file_name}.py")
+        #         with open(file_path, "w", encoding="utf-8") as fid:
+        #             if additional_imports is not None:
+        #                 library_imports = additional_imports["library_imports"]
+        #                 for library_import in library_imports:
+        #                     fid.write(f"{library_import}\n")
+        #                 fid.write("\n")
+        #                 class_name_with_parent = additional_imports["class_name_with_parent"]
+        #                 fid.write(f"class {class_name_with_parent}:\n")
+        #             else:
+        #                 fid.write(f"class {class_name}:\n")
+        #             methods_structure = []
+        #             for initial_command_name in method_list:
+        #                 if initial_command_name in SKIP_XML:
+        #                     continue
+        #                 command_obj = command_map[initial_command_name]
+        #                 python_method = command_obj.to_python(custom_functions, prefix='    ')
+        #                 methods_structure.append(name_map[initial_command_name])
+        #                 all_commands.append(initial_command_name)
+        #                 fid.write(f"{python_method}\n")
+        #         fid.close()
+        #         try:
+        #             subprocess.run(["python", str(file_path)])
+        #         except:
+        #             print(python_method)
+        #             raise RuntimeError(f"Failed to execute {file_path}") from None
+
+        #         class_structure[file_name] = [class_name, methods_structure]
+        #     package_structure[module_name] = class_structure
+
+    for command_name in name_map.keys():
+        if command_name not in all_commands:
+            print(f"{command_name} is not in the structure map")
+
+    write_global__init__file(library_path, module_name_list)
+    write__init__file(library_path)
+
+    print(f"Commands written to {library_path}")
```

Comment:
I rather you do not use ``print``. Please use ``logging``, for this case, you can use `loglevel=INFO`.